metrics/sqlite_query.py

```diff
@@ -0,0 +1,56 @@
+import sys
+import logging
+from sqlalchemy import create_engine, or_
+from sqlalchemy.orm import sessionmaker
+from db.models import Base, Issue, IssueState
+
+
+class SQLiteQuery:
+
+    def __init__(self, database) -> None:
+        self.logger = logging.getLogger(self.__module__)
+        self.logger.setLevel(logging.DEBUG)
+        handler = logging.StreamHandler(sys.stdout)
+        handler.setLevel(logging.DEBUG)
+        formatter = logging.Formatter(
+            '%(asctime)s - %(name)s - %(levelname)s - %(message)s')
+        handler.setFormatter(formatter)
+        self.logger.addHandler(handler)
+        engine = create_engine(f"sqlite:///{database}")
+        Base.metadata.create_all(engine, Base.metadata.tables.values(), checkfirst=True)
+        session_maker = sessionmaker(bind=engine)
+        self.session = session_maker()
+
+    def _check_session(self):
```

Comment:
I think this is overkill , if you don't have a session you will get exception in any case . Also you will get exception if something goes wrong during session creation . So this exception will dead code which will never raised